src/rules.js

```diff
@@ -2,25 +2,27 @@
 
 var fs = require('fs');
 var path = require('path');
-var rules; // Cashing list of rules, so that we only load them once
```

Comment:
The function would generate different results depending on the value of `additionalRulesDir`, however the value  `additionalRulesDir` shouldn't change during a single call to gherkin lint. Either way I think it's good you removed the caching, because it makes things harder to break in the future and i agree that it's probably not very expensive to load the rules files twice, given the number of existing rules.